app.js

```diff
@@ -1,11 +1,82 @@
 /* Imports */
 
 /* Get DOM Elements */
+const cityArticle = document.getElementById('city-article');
+
+const nameInput = document.getElementById('name-input');
+const nameDisplay = document.getElementById('name-display');
+const climateSelect = document.getElementById('climate-select');
+const climateDisplay = document.getElementById('climate-image');
+const architectureSelect = document.getElementById('architecture-select');
+const architectureDisplay = document.getElementById('architecture-image');
+const attractionInput = document.getElementById('attraction-input');
+const addButton = document.getElementById('add-button');
+const attractionList = document.getElementById('attraction-list');
 
 /* State */
+let city = {
+    name: 'Cool Town',
+    climate: 'desert',
+    architecture: 'modern',
+    attractions: [],
+};
+
+/* Helper Functions */
+function pageLoad() {
+    displayControls();
+    displayCity();
+    displayAttractions();
+}
```

Comment:
good job dividing the page render into three different functions so you can just render a part of it when state changes — this is in a nutshell what React does under the hood so it's good to have a solid grasp on this concept